src/common-lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+export enum ENTRANCE {
+  ADMIN = 'admin-self-add',
```

Comment:
let's use the value: `circle-create-initial-admin`
```suggestion
  ADMIN = 'admin-self-add',
  ADMIN = 'circle-create-initial-admin',
```